pkg/monitortestlibrary/platformidentification/types.go

```diff
@@ -283,3 +283,22 @@ func getArchitecture(clientConfig *rest.Config) (string, error) {
 
 	return "amd64", errors.New("could not determine architecture from master nodes")
 }
+
+// IsPlatformNamespace is a utility to detect if the namespace is considered platform
+func IsPlatformNamespace(nsName string) bool {
+	switch {
+	case nsName == "default" || nsName == "kubernetes" || nsName == "openshift":
+		return true
+
+	case strings.HasPrefix(nsName, "openshift-must-gather-"):
```

Comment:
`openshift-debug` has the same issue as well https://github.com/openshift/oc/blob/d6387d4acdcc020925d993d896437cffbf883671/pkg/cli/debug/debug.go#L1271

Reply:
updated.

Reply:
Thank you. nit:
```suggestion
	case strings.HasPrefix(nsName, "openshift-must-gather-") || strings.HasPrefix(nsName, "openshift-debug-"):
```